types/keep-network__tbtc.js/src/CommonTypes.d.ts

```diff
@@ -0,0 +1,118 @@
+/// <reference types="node" />
+import type { Config as ElectrumConfig } from "./lib/ElectrumClient";
+import type { FoundTransaction, BitcoinNetworkType } from './BitcoinHelpers';
+import BN = require("bn.js");
+import * as net from 'net';
+export class Web3 {
+    constructor(provider: any, net?: net.Socket);
+
+    static modules: any;
+    readonly givenProvider: any;
+    defaultAccount: string | null;
+    defaultBlock: string | number;
+    readonly currentProvider: any;
+    setProvider(provider: any): boolean;
+    BatchRequest: new () => any;
+    static readonly providers: any;
+
+    utils: any;
+    eth: any;
+    shh: any;
+    bzz: any;
+    version: string;
+    static readonly version: string;
+    static readonly utils: any;
+    extend(extension: any): any;
+}
+export type AbiType = 'function' | 'constructor' | 'event' | 'fallback';
+export type StateMutabilityType = 'pure' | 'view' | 'nonpayable' | 'payable';
+export interface AbiInput {
+    name: string;
+    type: string;
+    indexed?: boolean;
+	components?: AbiInput[];
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/keep-network__tbtc.js/src/CommonTypes.d.ts;
  Line: 33;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.